pom.xml

```diff
@@ -67,15 +67,20 @@
         <httpcomponents.version>4.3.3</httpcomponents.version>
         <javassist.version>3.19.0-GA</javassist.version>
         <javax-activation.version>1.1.1</javax-activation.version>
+        <jersey.version>2.25</jersey.version>
```

Comment:
Please can you make sure the NOTICES file is updated to include any new dependencies and these new versions.